src/wp-includes/post.php

```diff
@@ -6862,6 +6862,11 @@ function wp_mime_type_icon( $mime = 0, $preferred_ext = '.png' ) {
 		$icon = wp_cache_get( "mime_type_icon_$mime" );
 	}
 
+	// Check preferred file format is available and if it is begins with a period or not.
```

Comment:
I think this is a better comment:

// Check if preferred file format variable is present and is a valid file extension.

Reply:
Maybe more like "validly formatted file extension", since we're not actually checking the extension validity.